.gitignore

```diff
@@ -81,4 +81,7 @@ qtcreator-*
 .idea
 
 # FreeCAD backup files
-*.fcstd1
\ No newline at end of file
+*.fcstd1
+
+.DS_Store
+robert_gazebo/meshes/.DS_Store
```

Comment:
these is no need to add .DS_Store ignore in each folder. Rule in row no 86 will ignore it in every folder.